CONTRIBUTING.md

```diff
@@ -2,12 +2,32 @@
 
 Everybody is invited and welcome to contribute to Home Assistant. There is a lot to do...if you are not a developer perhaps you would like to help with the documentation on [home-assistant.io](https://home-assistant.io/)? If you are a developer and have devices in your home which aren't working with Home Assistant yet, why not spent a couple of hours and help to integrate them? 
 
+## Getting Started
 The process is straight-forward.
-
  - Fork the Home Assistant [git repository](https://github.com/home-assistant/home-assistant).
  - Write the code for your device, notification service, sensor, or IoT thing.
  - Ensure tests work.
  - Create a Pull Request against the [**dev**](https://github.com/home-assistant/home-assistant/tree/dev) branch of Home Assistant.
 
-Still interested? Then you should take a peak at the [developer documentation](https://home-assistant.io/developers/) to get more details. 
```

Comment:
We would like to keep this file as short as possible, to only have one source to update all the time.

@fabaff spent some cycles moving everything to the home-assistant.io site and I suggest you rather add more content there